Source/OnlineSubsystemEpic/Private/OnlinePrensenceEpic.h

```diff
@@ -36,3 +36,5 @@ class FOnlinePresenceEpic
 
 	virtual EOnlineCachedResult::Type GetCachedPresenceForApp(const FUniqueNetId& LocalUserId, const FUniqueNetId& User, const FString& AppId, TSharedPtr<FOnlineUserPresence>& OutPresence) override;
 };
+
+typedef TSharedPtr<class FOnlinePresenceEpic, ESPMode::ThreadSafe> FOnlinePresenceEpicPtr;
```

Comment:
Please add a newline to the end of files. This is needed for UNIX compatability.